scripts/pull_github_pr.sh

```diff
@@ -85,14 +70,18 @@ if [[ $nr_commits == 1 ]]; then
 		fi
 	fi
 	git commit --amend -m "${message}${closes}"
+	gh pr edit "${PR_NUM}" -b "${message}${closes}"
+	gh pr checkout "${PR_NUM}"
+	git rebase next
+	git push -f
```

Comment:
Would be better to fully qualify this push, i.e. provide the remote branch name. Are we pushing to `next` or the PR branch here?

Reply:
The push is to the PR branch, I will make it more explicit

Reply:
Fixed